dart/Dockerfile

```
@@ -1,15 +1,25 @@
-FROM dart:3.2
+FROM dart:3.2 AS build
 
+# Set the working directory
 WORKDIR /app
 
-ADD pubspec.yaml /app/
-
+# Add pubspec.yaml and get dependencies
+ADD pubspec.* ./
 RUN dart pub get --no-precompile
 
-ADD . /app/
-
+# Copy app source code and AOT compile it
+ADD . .
```

```suggestion
{{#files}}
  COPY '{{source}}' '{{target}}'
{{/files}}
```

this will copy all dart files (with the modification below on `dart/config.yaml`)